Resources/Server Info/RP_Rules.txt

```diff
@@ -1,103 +1,119 @@
+    - Names of notable famous or fictional persons or names that resemble/parody them are strictly forbidden. You are not clever if you slightly change a famous name around. Terrible names open you up to being politely reminded to change it, smited, or instantly banned, depending on severity.
+
+[color=#a4885c]10.[/color] Act like an actual human being on a space station in a low-roleplay (LRP) environment. Do not use text speak or emoticons IC, and do not refer to OOC things like admins in-game.
+
+[color=#a4885c]11.[/color] Don't be a dick. You are playing a multiplayer game with other people who also want to enjoy the game.
+    - Do not intentionally make other players' lives hell for your own amusement.
+    - Antagonists have lots of leeway with this rule and may kill/sabotage as they see fit, but if it degrades the experience for majority of the rest of the server you will be told to stop.
+    - [color=#ff0000]THE ROUND IS NOT OVER UNTIL THE END-ROUND SUMMARY APPEARS. KILLING SOMEONE FOR NO REASON BEFORE THIS WILL BE HANDLED ACCORDINGLY.[/color]
+
+[color=#a4885c]12.[/color] Don't harass or target players across rounds for actions in prior rounds or for actions outside of the game (this is referred to as "Metagrudging".) 
+    - Annoying players for IC reasons in the current round is fine; doing it across rounds or as a ghost role after they kill you is not.
+
+[color=#a4885c]13.[/color] Don't use information gained from outside your character's knowledge to gain an advantage (this is referred to as "Metagaming").
+    - Using information you gain from outside your own character (such as spectating while a ghost, through screenshots or metacomms, or other means) to your advantage is strictly forbidden. You remember all events up until you fall unconscious, even after cloning. If you take a ghost role, unless otherwise stated, you DO NOT REMEMBER ANYTHING from your past life.
+
+[color=#a4885c]14.[/color] Follow escalation rules and don't murder someone for slipping you, use common sense.
+    - You can defend yourself to the extent of protecting your own life.
+    - Security is unique in escalation and can escalate to less lethal to arrest criminals as needed.
+    - Don't outright leave people to die if you get in a fight, make an effort to heal them or bring them to Medbay. A
+    - Adminhelp the situation if you think someone is over-escalating.
+    - Department strikes, revolutions (ex: cargonia and any variation thereof), or cults are antagonist-only activites and are forbidden.
+
+[color=#a4885c]15.[/color] Don't immediately ghost or suicide from your role if you do not get antagonist (referred to as "Antag-rolling").
+    - This is not fair to other players actually waiting patiently for an antagonist round. Alternatively, if you do not want to play an antagonist or do not want to cause conflict (ex: friendly antagonists), do not opt-in for antagonist roles.
```

Comment:
Does this imply friendly antagonists are against the rules? I think this might be a bit excessive: what if a traitor wanted to pull up a merchant gimmick? (I guess this classifies as a neutral antagonist rather than a friendly one?). I think the "do not want to cause conflict" clarifies the point enough and the example just makes it more ambiguous.